platform/viewer/public/config/default.js

```diff
@@ -1,72 +1,140 @@
-window.config = {
-  // default: '/'
-  routerBasename: '/',
-  extensions: [],
-  showStudyList: true,
-  filterQueryParam: false,
-  servers: {
-    dicomWeb: [
-      {
-        name: 'DCM4CHEE',
-        wadoUriRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/wado',
-        qidoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
-        wadoRoot: 'https://server.dcmjs.org/dcm4chee-arc/aets/DCM4CHEE/rs',
-        qidoSupportsIncludeField: true,
-        imageRendering: 'wadors',
-        thumbnailRendering: 'wadors',
-        enableStudyLazyLoad: true,
+window.config = ({ servicesManager, dependencies }) => {
+  const { SimpleDialog, merge } = dependencies;
```

Comment:
@dannyrb maybe we can find better ways to pass in components/deps than this one? I'm all :ear: